src/cloud/project/project-routes-more-redir.md

```diff
@@ -12,6 +12,10 @@ The following demonstrates how to manage redirection rules on your {{site.data.v
 
 {% include cloud/note-route-all-placeholder.md %}
 
+{:.bs-callout-warning}
+In Magento Commerce Cloud, large numbers of non-regex redirects/rewrites should not be done at the Nginx layer as it can cause performance issues. If the size of your `routes.yaml` file is 32KB or larger, you should offload your non-regex redirects/rewrites to Fastly.
+The article [Offload non-regex redirects to Fastly instead of Nginx](https://support.magento.com/hc/en-us/articles/360035192891-Offload-non-regex-redirects-to-Fastly-instead-of-Nginx-routes-) explains how this can be achieved.
+
```

Comment:
```suggestion
{:.bs-callout-warning}
For {{ site.data.var.ece }} projects, configuring large numbers of non-regex redirects and rewrites in the `routes.yaml` file can cause performance issues. If your `routes.yaml` file is 32KB or larger, offload your non-regex redirects and rewrites to Fastly. See [Offload non-regex redirects to Fastly instead of Nginx (routes)](https://support.magento.com/hc/en-us/articles/360035192891-Offload-non-regex-redirects-to-Fastly-instead-of-Nginx-routes-) in the _Magento Help Center_.
```